packages/embed/src/setups/createElement.js

```diff
@@ -0,0 +1,24 @@
+export default function createElement(tagName, attributes, ...children) {
+  const element = document.createElement(tagName);
+
+  Object.keys(attributes).forEach(name => {
+    const value = attributes[name];
+
+    if (/^on[A-Z]/.test(name)) {
+      element.addEventListener(name.substr(2).toLowerCase(), value.bind(element));
+    } else if (name === 'style') {
+      const styles = value;
+      const elementStyle = element.style;
+
+      Object.keys(styles).forEach(name => {
+        elementStyle[name] = styles[name];
+      });
+    } else if (typeof value !== 'undefined') {
+      element.setAttribute(name, value);
```

Comment:
This does not seem to take [boolean attributes](https://www.w3.org/TR/html52/infrastructure.html#sec-boolean-attributes) into account. e.g. `disabled`, `checked`, etc...

Reply:
Fixed. At some points, I thought your wrote something similar before.